bottlecap/src/proc/mod.rs

```diff
@@ -180,6 +211,135 @@ fn get_uptime_from_path(path: &str) -> Result<f64, io::Error> {
     ))
 }
 
+#[must_use]
+pub fn get_fd_max_data(pids: &[i32]) -> f64 {
+    get_fd_max_data_from_path(PROC_PATH, pids)
+}
+
+fn get_fd_max_data_from_path(path: &str, pids: &[i32]) -> f64 {
+    let mut fd_max: f64 = 1024.0; // Default to hard limit set by AWS Lambda
```

Comment:
maybe move the `1024` to be a constant?

Reply:
also, why float? The fd number should always be an int, right?

Reply:
True it is always an int, I made it a float to be consistent with all the other metrics